src/Mvc/Mvc.Abstractions/src/ModelBinding/CompositeBindingSource.cs

```diff
@@ -89,7 +89,7 @@ public class CompositeBindingSource : BindingSource
         /// <inheritdoc />
         public override bool CanAcceptDataFrom(BindingSource bindingSource)
         {
-            if (bindingSource == null)
+            if (bindingSource is null)
```

Comment:
It would help if you keep focus on this already big PRs. Small changes like this distract and don't add value individually. (If we decided to convert the entire repo to use this style then so be it, but there's no agreement on this, and it would be more efficient done in a single pass).

Reply:
The compiler was complaining because the type has an equality operator. LHS is a non-null type, and RHS is null, so it says the two can never be equal. It was weird, but I didn't want to spend too much time on this